### PR TITLE
Fix langs + add de and fr langs 

### PR DESCRIPTION
**Pull request by Matho**
Hello

I fix some langs problems + I add new langs. That lang files comes from discussion at http://drupal.org/node/1278662 

Martin

Edit: I add small fix for Opera browser

### CODE REVIEW

**Comment by timohausmann** [comment]
Thanks for the de.js

File: css/codemirror.css [hunk 1]
```diff
@@ -1,6 +1,7 @@
 .CodeMirror {
   overflow: auto;
-  height: 90%;
```
Comment (tinymce-plugins) [inline css/codemirror.css:3]: Is this an unrelated change?
